setup.sh

```diff
@@ -255,9 +255,15 @@ install_deps() {
 
     # Need to install pnpm first before run `make install_deps`
     # in webapp.
+    # Following: https://pnpm.io/installation#using-corepack
     echo "Installing pnpm"
     if ! which pnpm >/dev/null 2>&1; then
-        curl -fsSL https://get.pnpm.io/install.sh | env PNPM_VERSION=10.0.0 sh -
+        if [[ -n "${IS_MAC}" ]]; then
```

Comment:
```suggestion
        if [ -n "${IS_MAC}" ]; then
```
I know this runs under `bash` but might as well stay sh-compatible when it's easy.